src/components/Homepage/Topics.js

```diff
@@ -0,0 +1,27 @@
+function Topics() {
+  return (
+    <article className="py-4 px-[25%]">
+      <nav className="flex flex-col-reverse md:flex-row md:items-center md:justify-center ">
+        <div className="mb-4 border-b border-[#5cb85c] dark:border-[#f2f2f2] flex-grow">
+          <ul className="flex flex-wrap -mb-px text-sm font-medium text-center">
+            <li className="mr-2">
+              <button className="inline-block p-4 rounded-t-lg border-b-2 border-[#5cb85c] text-[#5cb85c] hover:text-[#f2f2f2] hover:border-[#5cb85c]">
+                Global Feed
+              </button>
+            </li>
+          </ul>
+        </div>
+        <div className="md:ml-6">
```

Comment:
The Tags section should be its own component, its not very tied into the Feed. I would also not have it be part of this component.

Reply:
To clarify, would the "tags" section be the "Global feed" tab and the input search bar be its own component?


<img width="906" alt="Screen Shot 2022-07-02 at 2 15 09 PM" src="https://user-images.githubusercontent.com/60503218/177012013-0ccf63dd-10ab-4381-a3e6-2b25ab50bff9.png">

Reply:
Popular Tags should not be an input, it will have tags that can be enabled or disabled in the future. Feed (which global feed will be under) should be separate from popular tags component.

Reply:
I will change it from an input html block into a list html block